doc/user_guide/user_guide.md

```diff
@@ -43,7 +43,69 @@ We recommend the following requirements for the VM instance:
 | RAM  | 2GB     | 8GB         |
 | CPU  | 1 Core  | 2 Cores     |
 
-### Password
+### Use the AMI
+
+#### Overview
+
+1. Create a Security Group with open inbound ports for `ssh` (22) and `Jupyter` (8888)
```

Comment:
are you sure, you want to expose jupyter to the world

Reply:
Added a notice in the details about the security risk